lib/appsignal.rb

```diff
@@ -301,6 +294,15 @@ def start_stdout_logger
         "appsignal: #{msg}\n"
```

Comment:
@thijsc I kept this intact, but wouldn't it be more useful to use the normal `log_formatter` here too?